### DISCUSSION TITLE
Config checking

### DISCUSSION THREAD

**Discussion by WilsontheWolf** [renovatebot/renovate#15582] [Request Help]
Hello there. I was checking my config. I've modified the original one, and now the bot doesn't say which pr's it will make. I wanted to make sure that its working properly. Repo: https://github.com/IIoW/IIslander/pull/229

Specifically I wanted to make sure that:
- My action pr's were merged into one
- Docker examples are not being run

Thanks!

**Comment by rarkins** [comment]
There is a "What to expect" section which says which PRs it will make. Do you mean that it's not been updated since you made changes? 

You can log in at app.renovatebot.com to see if any job has run and what the logs are from that

**Reply by WilsontheWolf** [reply]
Yes correct, the bot has not updated that section since I made changes.
